jlink.go

```diff
@@ -155,19 +155,26 @@ func (j *JLink) buildCustomJRE(layerPath string) error {
 func (j *JLink) validArgs() bool {
 	jlinkArgs := j.Args[:0]
 	var skipNext, modsFound bool
+	lowerNext := true
 	for _, a := range j.Args {
 		if skipNext {
 			skipNext = false
 			continue
 		}
-		a = strings.ToLower(a)
+		if lowerNext {
+			a = strings.ToLower(a)
+		} else {
+			lowerNext = true
+		}
+
```

Comment:
Do you think we should actually be lowercasing these values? If someone passes an argument though, we should probably honor the way it's presented exactly. I'm curious about what you think.

To me, it seems like we are probably lowercasing the value here because we need to compare and pick out the `--output` argument. We don't want someone to put `--Output` and bypass that. It might be easier to just do the lowercase for the purpose of comparison but always put the original value into `jlinkArgs`.

Reply:
ah, that's an interesting option, to just lowercase during the comparison and keep the original value as is; I was focusing strictly on the `--add-modules` argument.

Let me update the PR with this suggestion!